docs/getting-started/istio/index.md

```diff
@@ -55,7 +55,7 @@ spec:
   hosts:
   - rollouts-demo.local
   http:
-  - name: primary  # Should match spec.strategy.canary.stableSertrafficRouting.istio.virtualService.routes
+  - name: primary  # Should match spec.strategy.canary.trafficRouting.istio.virtualService.routes
```

Comment:
This is by far the oddest change in the PR

Reply:
🤣 looks like I wanted to say stableService, then had a mild stroke before remembering it should be trafficRouting.